packages/waku/src/config.ts

```diff
@@ -49,14 +49,16 @@ export interface Config {
    */
   middleware?: () => Promise<{ default: Middleware }>[];
   /**
-   * Enhander for Hono
+   * Enhancer for Hono
    * Defaults to `undefined`
    */
-  unstable_honoEnhancer?:
-    | (<Hono>(createApp: (app: Hono) => Hono) => (app: Hono) => Hono)
-    | undefined;
+  unstable_honoEnhancer?: unstable_HonoEnhancer | undefined;
 }
 
+export type unstable_HonoEnhancer<Hono = never, MiddlewareHandler = never> = (
```

Comment:
I would like to avoid exporting type from this file. If you need it in the plugin, simply define it there.

Reply:
Yes, that sounds good.